taskvine/src/examples/vine_example.py

```diff
@@ -59,13 +59,13 @@
       # Note that when adding a file, we have to name its local name
       # (e.g. gzip_path), and its remote name (e.g. "gzip"). Unlike the
       # following line, more often than not these are the same.
-      t.add_file(gzip_path, "gzip", VINE_INPUT, cache=True)
+      t.add_input_file(gzip_path, "gzip", flags=VINE_UNPACK, cache=True)
```

Comment:
I don't think you need any extra flags here. In all your changes in this example, I don't see any need to use VINE_UNPACK.